src/ContainerRenameModal.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Button,
+    Form, FormGroup,
+    Modal, TextInput
+} from '@patternfly/react-core';
+import cockpit from 'cockpit';
+
+import * as client from './client.js';
+
+import { ErrorNotification } from './Notification.jsx';
+import * as dockerNames from 'docker-names';
+
+const _ = cockpit.gettext;
+
+class ContainerRenameModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: dockerNames.getRandomName(),
+            nameError: "",
+        };
+
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleRename = this.handleRename.bind(this);
+    }
+
+    componentDidMount() {
+        this._isMounted = true;
```

Comment:
`_isMounted` is not used anywhere. Please drop this function and the assignment in WillUnmount().

Reply:
Fixed in commit https://github.com/cockpit-project/cockpit-podman/commit/0c3706e1009fabc1fee2a17b247175993469a9c4